packages/neuron-wallet/src/startup/sync-block-task/task.ts

```diff
@@ -22,7 +22,7 @@ const testIndexer = async (url: string): Promise<boolean> => {
 const run = async () => {
   DatabaseInitSubject.getSubject().subscribe(async (params: DatabaseInitParams) => {
     const { network, genesisBlockHash, chain } = params
-    if (network && genesisBlockHash.startsWith('0x')) {
+    if (network && (genesisBlockHash.startsWith('0x') || genesisBlockHash === 'stop')) {
```

Comment:
Why it should go through to start the queue when `stopped`?

Reply:
The stop queue logic is in ` syncSwitchNetwork`.